sources/packages/web/src/main.ts

```diff
@@ -152,6 +138,8 @@ AppConfigService.shared.init().then(() => {
     .component("ToggleContent", ToggleContent)
     .component("FormioContainer", FormioContainer)
     .component("FooterButtons", FooterButtons)
-    .directive("tooltip", Tooltip)
-    .mount("#app");
+    .directive("tooltip", Tooltip);
+
+  app.config.globalProperties.emitter = emitter;
```

Comment:
Does it need to be set here? My understanding is that we need have this object shared and it can be achieved directly in the composable. I do not see a reason to have it declared here since we are not accessing the Vue properties directly. Please let me know if I am missing something.

Reply:
as discussed updated  composable